src/PrestaShopBundle/Resources/config/services/bundle/translation.yml

```
@@ -2,111 +2,80 @@ services:
     _defaults:
         public: true
 
-    #TRANSLATIONS FACTORIES
-    prestashop.translation.translations_factory:
-        class: PrestaShopBundle\Translation\Factory\TranslationsFactory
-
-    ps.translations_factory:
-        alias: "prestashop.translation.translations_factory"
-
-    prestashop.translation.theme_translations_factory:
-        class: PrestaShopBundle\Translation\Factory\ThemeTranslationsFactory
-        arguments: ['@prestashop.translation.theme_provider']
-        calls:
-            - [ addProvider, ['@prestashop.translation.theme_provider']]
-
-    ps.theme_translations_factory:
-        alias: "prestashop.translation.theme_translations_factory"
-
-    #TRANSLATIONS PROVIDERS
-    prestashop.translation.backoffice_provider:
-        class: PrestaShopBundle\Translation\Provider\BackOfficeProvider
+    prestashop.translation.back_provider_factory:
```

If we rename services, we should keep the same naming as what has been introduced with grid
`prestashop.core.grid.factory.backup`
as an example it could be
```suggestion
    prestashop.translation.provider.factory.back:
```